nipype/interfaces/afni/utils.py

```diff
@@ -1377,6 +1380,9 @@ class MaskToolInputSpec(AFNICommandInputSpec):
              'or using the labels in {R,L,A,P,I,S}.',
         argstr='-fill_dirs %s',
         requires=['fill_holes'])
+    verbose = traits.Int(
```

Comment:
Note that you have `verbose` in `Copy`. If it's okay to mix-and-match (i.e., there are some interfaces that already have `verbose`), I'd prefer that you use `verbose` for all new additions. If this would be the first interface with `verbose`, I'd say switch it to `verb` for consistency.

Reply:
Sorry I am not sure to follow. I added `verbose` to `Allineate`, `Copy`, `MaskTool` and`Tcat`. These interfaces had no verbosity parameter before. I didn't touch the other ones which had already a `verb` or a `verbose` parameter as you told me.

Reply:
Sorry, I think I wrote this just as you made those changes. I wasn't sure if you were going to keep it at `verb`, but the changes you made look good. You can safely ignore that last message. (Also this one.)